pkg/formatting/errors.go

```diff
@@ -0,0 +1,46 @@
+package formatting
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+func HumanizeJSONErr(jsonContent string, err error) error {
```

Comment:
the input is in yaml the transformed content is in json which is fine it's a subset but the lines are not gong to match, right?

Reply:
Oof... Yeah I did not consider that. Let me add a test for yaml to see how that behaves. Thanks for catching